app/assets/javascripts/backbone/views/timeline_table_view.coffee

```diff
@@ -22,10 +22,10 @@ class MS.Views.TimelineTableView extends Backbone.View
     'click .timeline-action[data-action="new-step"]': 'newStep'
     'click #project-group-menu [data-action="add-child-group"]': 'newChildGroup'
     'click #project-group-menu [data-action="add-child-step"]': 'newChildStep'
-    'click #project-group-menu [data-action="edit"]': 'editGroup'
+    'click .project-groups[data-action="edit"]': 'editGroup'
     'confirm:complete #project-group-menu [data-action="delete"]': 'deleteGroup'
     'click .project-step': 'showStep'
```

Comment:
This would no longer work given the new class name. It is also no longer needed.